python/nav/ipdevpoll/neighbor.py

```diff
@@ -112,20 +111,20 @@ def __init__(self, record, local_address=None):
         if local_address:
             self._invalid_neighbor_ips.append(str(local_address))
 
-        self.netbox = self.interface = None
```

Comment:
Yes, but there is a difference between seeing three separate neighbor records on one port for three different neighboring ports, and matching a single port identity to three different ports from one single neighbor record.

Please drop by me for a chat on this :)